x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/integration/DocumentLevelSecurityTests.java

```diff
@@ -116,7 +117,8 @@
             InternalSettingsPlugin.class,
             DenseVectorPlugin.class,
             SpatialPlugin.class,
-            PercolatorPlugin.class
+            PercolatorPlugin.class,
+            MapperExtrasPlugin.class
```

Comment:
Is this necessary for this PR?

Reply:
Nah removed from this PR => will fix separately :)